velox/dwio/common/compression/CMakeLists.txt

```diff
@@ -12,5 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_library(velox_dwio_common_compression LzoDecompressor.cpp)
+if(${VELOX_BUILD_TESTING})
+  add_subdirectory(tests)
+endif()
+
+add_library(velox_dwio_common_compression Compression.cpp LzoDecompressor.cpp)
```

Comment:
Does this need to be under dwio? Looks like it will be used for non-DWIO code. Maybe move it somewhere else. It would be nice to do this refactoring in a separate PR.

Reply:
Where would you recommend to put it? Maybe `velox/common/compression` or `velox/compression`? I can help refactor it in a separate  PR.

Reply:
velox/common/compression sounds good. Thanks.

Reply:
@jinchengchenghh I think you have moved to the wrong location: it should be velox/common/compression/Compression.h but not velox/dwio/common/compression/Compression.h? @mbasmanova

Reply:
I will refactor in a separate PR as this discussion suggested. https://github.com/facebookincubator/velox/pull/5544#discussion_r1265721028

Reply:
@jinchengchenghh Thank you. Just to clarify, we'll wait for a refactoring PR to proceed.